hat/assets/js/apps/Iaso/domains/orgUnits/components/OrgUnitInfosComponent.js

```diff
@@ -307,13 +361,31 @@ const OrgUnitInfosComponent = ({
         initialFormState(orgUnit, instanceId),
     );
 
+    const [isSaveDisabled, setSaveDisabled] = useState(true);
+
+    const showSpeedDialInstanceActions =
+        orgUnit.reference_instance ||
+        (formId === referenceFormId &&
+            formId !== undefined &&
+            referenceFormId !== undefined);
+
+    const isNewOrgunit = params.orgUnitId === '0';
+
+    useEffect(() => {
+        if (
+            orgUnit.name.value === '' ||
+            orgUnit.org_unit_type_id.value === null
+        ) {
+            setSaveDisabled(true);
+        } else {
+            setSaveDisabled(false);
+        }
```

Comment:
`setSaveDisabled(orgUnit.name.value === '' || orgUnit.org_unit_type_id.value === null)` ? , not sure you need a state for this